Doc/whatsnew/3.8.rst

```diff
@@ -735,6 +735,13 @@ Added a new variable holding structured version information for the
 underlying ncurses library: :data:`~curses.ncurses_version`.
 (Contributed by Serhiy Storchaka in :issue:`31680`.)
 
+The extended color functions added in ncurses 6.1 will be used transparently
```

Comment:
This entry should be moved to `Doc/whatsnew/3.9.rst`.

Reply:
Fixed.